Tests/MongoSwiftSyncTests/SpecTestRunner/TestOperation.swift

```diff
@@ -599,122 +769,339 @@ struct RenameCollection: TestOperation {
+        throws -> TestOperationResult? {
         guard case let .database(database) = target else {
             throw TestError(message: "database not provided to listCollectionObjects")
         }
-        _ = try database.listMongoCollections(session: session)
+        _ = try database.listMongoCollections(session: nil)
         return nil
     }
 }
 
 struct ListCollectionNames: TestOperation {
-    func execute(on target: TestOperationTarget, session: ClientSession?) throws -> TestOperationResult? {
+    func execute(on target: TestOperationTarget, sessions _: [String: ClientSession])
+        throws -> TestOperationResult? {
         guard case let .database(database) = target else {
             throw TestError(message: "database not provided to listCollectionNames")
         }
-        return try .array(database.listCollectionNames(session: session).map { .string($0) })
+        return try .array(database.listCollectionNames(session: nil).map { .string($0) })
     }
 }
 
 struct Watch: TestOperation {
-    func execute(on target: TestOperationTarget, session: ClientSession?) throws -> TestOperationResult? {
+    func execute(on target: TestOperationTarget, sessions _: [String: ClientSession])
+        throws -> TestOperationResult? {
         switch target {
         case let .client(client):
-            _ = try client.watch(session: session)
+            _ = try client.watch(session: nil)
         case let .database(database):
-            _ = try database.watch(session: session)
+            _ = try database.watch(session: nil)
         case let .collection(collection):
-            _ = try collection.watch(session: session)
+            _ = try collection.watch(session: nil)
+        case .session, .testRunner:
+            break
         }
         return nil
     }
 }
 
 struct EstimatedDocumentCount: TestOperation {
-    func execute(on target: TestOperationTarget, session: ClientSession?) throws -> TestOperationResult? {
+    func execute(on target: TestOperationTarget, sessions _: [String: ClientSession])
+        throws -> TestOperationResult? {
         guard case let .collection(collection) = target else {
             throw TestError(message: "collection not provided to estimatedDocumentCount")
         }
-        return try .int(collection.estimatedDocumentCount(session: session))
+        return try .int(collection.estimatedDocumentCount(session: nil))
+    }
+}
+
+struct StartTransaction: TestOperation {
+    let options: TransactionOptions
+
+    init() {
+        self.options = TransactionOptions()
+    }
+
+    func execute(on target: TestOperationTarget, sessions _: [String: ClientSession])
+        throws -> TestOperationResult? {
+        guard case let .session(session) = target else {
+            throw TestError(message: "session not provided to startTransaction")
+        }
+        _ = try session.startTransaction(options: self.options)
```

Comment:
do we need the `_`? I thought this method returned nil already

Reply:
You're right, I've removed it.